starknet_py/serialization/factory.py

```diff
@@ -115,6 +115,8 @@ def serializer_for_type(cairo_type: CairoType) -> CairoDataSerializer:
                 for name, variant_type in cairo_type.variants.items()
             )
         )
```

Comment:
`serializer_for_type` is also missing a case for `TypeIdentifier`. Do you think it is intended?

Reply:
It seems to be intended. `TypeIdentifier` is used to handle nested structs during parsing abi. As the result of parsing, we get python class with all the information about a nested struct

Reply:
Ok, thanks for the clarification!